clab/clab.go

```diff
@@ -58,15 +59,24 @@ func WithTimeout(dur time.Duration) ClabOption {
 
 func WithRuntime(name string, d bool, dur time.Duration, gracefulShutdown bool) ClabOption {
 	return func(c *CLab) {
+		if name == "" {
```

Comment:
lets make a `switch` here

first order is the cli flag
then env var
default is docker runtime

Reply:
added in 91bcd27